library/common-library/templates/_license_secret.yaml.tpl

```
@@ -0,0 +1,22 @@
+{{/*
+Renders the license key secret if user has not specified a custom secret.
+*/}}
+{{- define "common.license.secret" }}
+{{- if not (include "common.license._customSecretName" .) }}
+
+{{/* Fail if licenseKey is empty and required */}}
+{{- if not (include "common.license._licenseKey" .) }}
+    {{- fail "You must specify a licenseKey or a customSecretName containing it" }}
+{{- end }}
```

Get rid of the empty lines that we have "for logic" and/or add the YAML start token (`---`)
```suggestion
{{- end }}
---
```

---

This is a nit. It only renders empty lines.

---

Ok, looks like it's safe to do: https://go.dev/play/p/rX5kp9Pp2bx

---

Some YAML parsers (not the golang one and not Helm) will have issues having empty lines because they detect it as a two document YAML with the first one empty.

---

Good point, I'll go for that.